my_gazebo_concert/src/solutions/gazebo.services

```diff
@@ -0,0 +1,13 @@
+# PRIORITIES
+#
+# Turtle pond has scheduler_msgs.Request.DEFAULT_PRIORITY, teleop has 
+# scheduler_msgs.Request.HIGH_PRIORITY. If you want to change the priority
+# override it as below for any of the services:
+#  overrides:
+#    priority: 1000
+
+- resource_name: concert_service_gazebo/gazebo_robot_world
+  overrides:
+    parameters: gazebo_concert/gazebo.parameters
```

Comment:
If you want to override this you should be able to create a copy of the file in this package. I believe that the first part of this "path" is thepackage name. So it would be `my_gazebo_concert/gazebo.parameters` and you'll want to put the file in a similar location so that it can be found.